.github/workflows/post-release-changes.yaml

```diff
@@ -0,0 +1,51 @@
+name: Post-release changes
+
+on:
+  release:
+    types: [published]
```

Comment:
This looks good. However, published releases cannot have `ref tag` as `master`. They need to be semantically versioned, which means `v<major>.<minor>.<patch>` always.

Reply:
I don't exactly follow what you mean/what you want me to do. Could you provide some more direction? Sorry!

Reply:
No problem.

Whenever we publish a new release, we create a semantic versioning tag for it (https://github.com/iter8-tools/iter8/tagshttps://github.com/iter8-tools/iter8/tags).

This tag has the <major>.<minor>.<patch> version numbers always. Thus, our current release is version 0.7.21 with major being 0 and minor being 7.

In other words, the tag will be `v0.7.21`. The tag will never equal `master` in this workflow.

Reply:
Ah I see, sorry. I forgot this piece of code existed. Should be fixed now!